routes/render_add_form.js

```diff
@@ -0,0 +1,8 @@
+const request = require('request');
+
+module.exports = (req, res) => {
+  res.render('add_event', {
+    eachPlaces: res.locals.places,
```

Comment:
Could you call this `places` instead? eachPlaces doesn't really make sense (sorry)

Reply:
you are forgiven :stuck_out_tongue_winking_eye: